### ISSUE TITLE
Can we implement a message notification feature?

### ISSUE THREAD

**Issue by SahSih** [RocketChat/Rocket.Chat.PWA#14]
I have done one notification web application base on Firebase and service worker. I am thinking if we can develop this feature on Rocket.chat. As a chat app, notification is a really helpful feature. 
What do you guys think about this feature?

**Comment by InsaneNaman** [comment]
Hello @SahSih,
Welcome to Rocket.Chat . If i am not wrong you are talking about web notification or push notification and yes we can implement but the current app is built on top of Polymer 2 and we are thinking to re-build from scratch using Angular 4. So,for now it won't worth much if you contribute to current app but in future  every contribution from your side will help us a lot. Repo will be updated as soon as gsoc starts. So keep checking for latest updates. 😄 

PS:Feel Free to look our other projects and to ask question here or over there.Also, join our community at [demo.rocket.chat](https://demo.rocket.chat) 😄.

**Comment by ajayns** [comment]
Yes @SahSih, but as @InsaneNaman said, this repo is no longer maintained as we shift to Angular for a PWA.

**Comment by SahSih** [comment]
@InsaneNaman @ajayns  I see, thanks for you guys help! ^_^

**Comment by InsaneNaman** [comment]
😄 @SahSih Your most welcome buddy. Looking forward to catch you at our [community](https://demo.rocket.chat)

**Comment by Sing-Li** [comment]
@SahSih - @InsaneNaman and @ajayns are correct!  

But the shift to Angular may happen very very soon, perhaps even before GSoC official results - with a surprise learning and professional development opportunity for everyone involved. Please keep an eye out on this repository  (and the community) !

**Comment by InsaneNaman** [comment]
That's great @Sing-Li  sir 😄

**Comment by SahSih** [comment]
@ajayns @Sing-Li  @InsaneNaman  Thanks for you guys response!
